docs/tutorials/index.rst

```diff
@@ -20,16 +20,8 @@ you'll learn how to perform the following tasks:
  Get Started with a :ref:`Trial API <high-level-apis>`
 *******************************************************
 
-+---------------------------------+--------------------------------------------------------------+
-| Title                           | Description                                                  |
-+=================================+==============================================================+
-| :doc:`pytorch-mnist-tutorial`   | Based on the `PyTorch MNIST example`_, this tutorial shows   |
-|                                 | you how to port a simple image classification model for the  |
-|                                 | MNIST dataset.                                               |
-+---------------------------------+--------------------------------------------------------------+
-| :doc:`tf-mnist-tutorial`        | The TensorFlow Keras Fashion MNIST tutorial describes how to |
-|                                 | port a ``tf.keras`` model to Determined.                     |
-+---------------------------------+--------------------------------------------------------------+
+To get started with a :ref:`Trial API <high-level-apis>`, visit the :ref:`pytorch-mnist-tutorial`.
```

Comment:
In this case, the "the" makes the link text part of the sentence and not just a location, so the text should be written out. (Or just remove the "the".)

```suggestion
To get started with a :ref:`Trial API <high-level-apis>`, visit the :ref:`PyTorch MNIST tutorial <pytorch-mnist-tutorial>`.
```

Reply:
good catch